lnrpc/lightning.proto

```diff
@@ -515,6 +515,13 @@ service Lightning {
     */
     rpc ListPermissions (ListPermissionsRequest)
         returns (ListPermissionsResponse);
+
+    /* lncli: `updatenodeannouncement`
```

Comment:
Wondering if we should put this on one of the "newer" sub RPC servers instead 🤔. We've been moving to (though not systematically yet) slowly deprecate this one in favor of some of the "smaller" more domain specific ones (like `walletrpc` as an example).

Reply:
perfect, i can move it to a new one. However, what would be the domain (naming) for the new one? I would create the new service only with this endpoint and migrate the other endpoints in another PR so it does not break anything